fenix/app/src/main/res/values/strings.xml

```diff
@@ -1812,7 +1812,7 @@
     <!-- Message of warning dialog if users have no device authentication set up -->
     <string name="credit_cards_warning_dialog_message">Set up a device lock pattern, PIN, or password to protect your saved credit cards from being accessed if someone else has your device.</string>
     <!-- Message of warning dialog if users have no device authentication set up -->
-    <string name="credit_cards_warning_dialog_message_2" tools:ignore="UnusedResources">Set up a device lock pattern, PIN, or password to protect your saved cards from being accessed if someone else has your device.</string>
+    <string name="credit_cards_warning_dialog_message_2" tools:ignore="UnusedResources">Set up a device lock pattern, PIN, or password to protect your saved payment methods from being accessed if someone else has your device.</string>
```

Comment:
When the string changes, except for some exceptions, the string ID should be updated so that our localization tool Pontoon notifies localizers. Note: this is not the case for localization comments.
More info here: https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#string-identifiers-ids